specification/socks5plus/SPEC.md

```diff
@@ -0,0 +1,28 @@
+# Kaazing SOCKS 5+ protocol
+
+## Abstract
+
+This document specifies an extended form of the standard SOCKS 5 protocol used for Enterprise Shield (a.k.a. reverse connectivity). This is sometimes referred to as SOCKS 5+.
+
+## Requirements
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC2119][].
+
+## The Protocol
+Kaazing SOCKS 5+ protocol conforms to [SOCKS 5] except that an extra ATYP (address type) is supported, as follows:
+
+URI address: X'FE'
```

Comment:
This is OK, but it would be nice if it was a little more close to the style of the RFC specification.  For example it wasn't clear to me until reading the scripts that the AYTP we are reserving is "0xFE" (The X was confusing)